plotly/plotlyfig_aux/handlegraphics/updateQuiver.m

```diff
@@ -88,6 +88,45 @@
 
 %-------------------------------------------------------------------------%
 
+%-quiver barbs-%
```

Comment:
Nice - could you throw this in an `if isHG2` block? The arrow heads work on pre-2014b so this might do something strange like double up the arrow head for earlier releases.